src/index.php

```diff
@@ -176,7 +170,6 @@
     array('^/upload', 'ROLE_user'),
     array('^/script', 'ROLE_system_admin'),
 );
-// BCrypt needs PHP 5.5 on server, so instead have added "composer require ircmaxell/password-compat". IJH 2015-09
```

Comment:
This comment is an artifact of when password-compat was needed.  The history of the change is no longer needed.